{
  "stats": {
    "suites": 1,
    "tests": 4,
    "passes": 0,
    "pending": 0,
    "failures": 4,
    "start": "2022-08-02T15:00:31.613Z",
    "end": "2022-08-02T15:00:48.381Z",
    "duration": 16768,
    "testsRegistered": 4,
    "passPercent": 0,
    "pendingPercent": 0,
    "other": 0,
    "hasOther": false,
    "skipped": 0,
    "hasSkipped": false
  },
  "results": [
    {
      "uuid": "21e65a84-2f34-4924-8805-8d0a85545758",
      "title": "",
      "fullFile": "cypress/integration/10.TestCoverage_spec.js",
      "file": "cypress/integration/10.TestCoverage_spec.js",
      "beforeHooks": [],
      "afterHooks": [],
      "tests": [],
      "suites": [
        {
          "uuid": "a39aed57-3607-4b66-b014-263b4adb40e5",
          "title": "10 - Desenvolva testes para atingir 90% de cobertura total da aplicação",
          "fullFile": "",
          "file": "",
          "beforeHooks": [],
          "afterHooks": [],
          "tests": [
            {
              "title": "Cobertura de \"Statements\" total deverá ser maior ou igual a 90.",
              "fullTitle": "10 - Desenvolva testes para atingir 90% de cobertura total da aplicação Cobertura de \"Statements\" total deverá ser maior ou igual a 90.",
              "timedOut": null,
              "duration": 4130,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.task('getCoverage', (0, _getId.getId)()).its('total.statements.pct', {\n  timeout: 0\n}).should('be.gte', 90.00);",
              "err": {
                "message": "CypressError: `cy.task('getCoverage')` failed with the following error:\n\n> Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_35_638Z-debug.log\n\n\nhttps://on.cypress.io/api/task",
                "estack": "CypressError: `cy.task('getCoverage')` failed with the following error:\n\n> Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_35_638Z-debug.log\n\n\nhttps://on.cypress.io/api/task\n    at http://localhost:43217/__cypress/runner/cypress_runner.js:167117:78\n    at tryCatcher (http://localhost:43217/__cypress/runner/cypress_runner.js:13022:23)\n    at Promise._settlePromiseFromHandler (http://localhost:43217/__cypress/runner/cypress_runner.js:10957:31)\n    at Promise._settlePromise (http://localhost:43217/__cypress/runner/cypress_runner.js:11014:18)\n    at Promise._settlePromise0 (http://localhost:43217/__cypress/runner/cypress_runner.js:11059:10)\n    at Promise._settlePromises (http://localhost:43217/__cypress/runner/cypress_runner.js:11135:18)\n    at _drainQueueStep (http://localhost:43217/__cypress/runner/cypress_runner.js:7729:12)\n    at _drainQueue (http://localhost:43217/__cypress/runner/cypress_runner.js:7722:9)\n    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:43217/__cypress/runner/cypress_runner.js:7738:5)\n    at Async.drainQueues (http://localhost:43217/__cypress/runner/cypress_runner.js:7608:14)\nFrom Your Spec Code:\n    at Context.eval (http://localhost:43217/__cypress/tests?p=cypress/integration/10.TestCoverage_spec.js:103:8)\n\nFrom Node.js Internals:\n  Error: Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\n  npm ERR! code ELIFECYCLE\n  npm ERR! errno 1\n  npm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\n  npm ERR! Exit status 1\n  npm ERR! \n  npm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\n  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n  \n  npm ERR! A complete log of this run can be found in:\n  npm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_35_638Z-debug.log\n  \n      at ChildProcess.exithandler (child_process.js:294:12)\n      at ChildProcess.emit (events.js:198:13)\n      at maybeClose (internal/child_process.js:982:16)\n      at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)",
                "diff": null
              },
              "uuid": "31a7c7ad-1e00-4ebf-931b-0eb437e235ad",
              "parentUUID": "a39aed57-3607-4b66-b014-263b4adb40e5",
              "isHook": false,
              "skipped": false
            },
            {
              "title": "Cobertura de \"Functions\" total deverá ser maior ou igual a 90.",
              "fullTitle": "10 - Desenvolva testes para atingir 90% de cobertura total da aplicação Cobertura de \"Functions\" total deverá ser maior ou igual a 90.",
              "timedOut": null,
              "duration": 4134,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.task('getCoverage', (0, _getId.getId)()).its('total.functions.pct', {\n  timeout: 0\n}).should('be.gte', 90.00);",
              "err": {
                "message": "CypressError: `cy.task('getCoverage')` failed with the following error:\n\n> Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_39_974Z-debug.log\n\n\nhttps://on.cypress.io/api/task",
                "estack": "CypressError: `cy.task('getCoverage')` failed with the following error:\n\n> Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_39_974Z-debug.log\n\n\nhttps://on.cypress.io/api/task\n    at http://localhost:43217/__cypress/runner/cypress_runner.js:167117:78\n    at tryCatcher (http://localhost:43217/__cypress/runner/cypress_runner.js:13022:23)\n    at Promise._settlePromiseFromHandler (http://localhost:43217/__cypress/runner/cypress_runner.js:10957:31)\n    at Promise._settlePromise (http://localhost:43217/__cypress/runner/cypress_runner.js:11014:18)\n    at Promise._settlePromise0 (http://localhost:43217/__cypress/runner/cypress_runner.js:11059:10)\n    at Promise._settlePromises (http://localhost:43217/__cypress/runner/cypress_runner.js:11135:18)\n    at _drainQueueStep (http://localhost:43217/__cypress/runner/cypress_runner.js:7729:12)\n    at _drainQueue (http://localhost:43217/__cypress/runner/cypress_runner.js:7722:9)\n    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:43217/__cypress/runner/cypress_runner.js:7738:5)\n    at Async.drainQueues (http://localhost:43217/__cypress/runner/cypress_runner.js:7608:14)\nFrom Your Spec Code:\n    at Context.eval (http://localhost:43217/__cypress/tests?p=cypress/integration/10.TestCoverage_spec.js:108:8)\n\nFrom Node.js Internals:\n  Error: Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\n  npm ERR! code ELIFECYCLE\n  npm ERR! errno 1\n  npm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\n  npm ERR! Exit status 1\n  npm ERR! \n  npm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\n  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n  \n  npm ERR! A complete log of this run can be found in:\n  npm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_39_974Z-debug.log\n  \n      at ChildProcess.exithandler (child_process.js:294:12)\n      at ChildProcess.emit (events.js:198:13)\n      at maybeClose (internal/child_process.js:982:16)\n      at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)",
                "diff": null
              },
              "uuid": "dce3e8ac-5b7f-44f4-b1ef-30f95c59e3ce",
              "parentUUID": "a39aed57-3607-4b66-b014-263b4adb40e5",
              "isHook": false,
              "skipped": false
            },
            {
              "title": "Cobertura de \"Branches\" total deverá ser maior ou igual a 90.",
              "fullTitle": "10 - Desenvolva testes para atingir 90% de cobertura total da aplicação Cobertura de \"Branches\" total deverá ser maior ou igual a 90.",
              "timedOut": null,
              "duration": 4195,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.task('getCoverage', (0, _getId.getId)()).its('total.branches.pct', {\n  timeout: 0\n}).should('be.gte', 90.00);",
              "err": {
                "message": "CypressError: `cy.task('getCoverage')` failed with the following error:\n\n> Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_44_172Z-debug.log\n\n\nhttps://on.cypress.io/api/task",
                "estack": "CypressError: `cy.task('getCoverage')` failed with the following error:\n\n> Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_44_172Z-debug.log\n\n\nhttps://on.cypress.io/api/task\n    at http://localhost:43217/__cypress/runner/cypress_runner.js:167117:78\n    at tryCatcher (http://localhost:43217/__cypress/runner/cypress_runner.js:13022:23)\n    at Promise._settlePromiseFromHandler (http://localhost:43217/__cypress/runner/cypress_runner.js:10957:31)\n    at Promise._settlePromise (http://localhost:43217/__cypress/runner/cypress_runner.js:11014:18)\n    at Promise._settlePromise0 (http://localhost:43217/__cypress/runner/cypress_runner.js:11059:10)\n    at Promise._settlePromises (http://localhost:43217/__cypress/runner/cypress_runner.js:11135:18)\n    at _drainQueueStep (http://localhost:43217/__cypress/runner/cypress_runner.js:7729:12)\n    at _drainQueue (http://localhost:43217/__cypress/runner/cypress_runner.js:7722:9)\n    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:43217/__cypress/runner/cypress_runner.js:7738:5)\n    at Async.drainQueues (http://localhost:43217/__cypress/runner/cypress_runner.js:7608:14)\nFrom Your Spec Code:\n    at Context.eval (http://localhost:43217/__cypress/tests?p=cypress/integration/10.TestCoverage_spec.js:113:8)\n\nFrom Node.js Internals:\n  Error: Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\n  npm ERR! code ELIFECYCLE\n  npm ERR! errno 1\n  npm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\n  npm ERR! Exit status 1\n  npm ERR! \n  npm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\n  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n  \n  npm ERR! A complete log of this run can be found in:\n  npm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_44_172Z-debug.log\n  \n      at ChildProcess.exithandler (child_process.js:294:12)\n      at ChildProcess.emit (events.js:198:13)\n      at maybeClose (internal/child_process.js:982:16)\n      at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)",
                "diff": null
              },
              "uuid": "c39fe6f0-f625-4733-a58a-9e7f1745835d",
              "parentUUID": "a39aed57-3607-4b66-b014-263b4adb40e5",
              "isHook": false,
              "skipped": false
            },
            {
              "title": "Cobertura de \"Lines\" total deverá ser maior ou igual a 90.",
              "fullTitle": "10 - Desenvolva testes para atingir 90% de cobertura total da aplicação Cobertura de \"Lines\" total deverá ser maior ou igual a 90.",
              "timedOut": null,
              "duration": 4050,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.task('getCoverage', (0, _getId.getId)()).its('total.lines.pct', {\n  timeout: 0\n}).should('be.gte', 90.00);",
              "err": {
                "message": "CypressError: `cy.task('getCoverage')` failed with the following error:\n\n> Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_48_277Z-debug.log\n\n\nhttps://on.cypress.io/api/task",
                "estack": "CypressError: `cy.task('getCoverage')` failed with the following error:\n\n> Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_48_277Z-debug.log\n\n\nhttps://on.cypress.io/api/task\n    at http://localhost:43217/__cypress/runner/cypress_runner.js:167117:78\n    at tryCatcher (http://localhost:43217/__cypress/runner/cypress_runner.js:13022:23)\n    at Promise._settlePromiseFromHandler (http://localhost:43217/__cypress/runner/cypress_runner.js:10957:31)\n    at Promise._settlePromise (http://localhost:43217/__cypress/runner/cypress_runner.js:11014:18)\n    at Promise._settlePromise0 (http://localhost:43217/__cypress/runner/cypress_runner.js:11059:10)\n    at Promise._settlePromises (http://localhost:43217/__cypress/runner/cypress_runner.js:11135:18)\n    at _drainQueueStep (http://localhost:43217/__cypress/runner/cypress_runner.js:7729:12)\n    at _drainQueue (http://localhost:43217/__cypress/runner/cypress_runner.js:7722:9)\n    at Async.../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:43217/__cypress/runner/cypress_runner.js:7738:5)\n    at Async.drainQueues (http://localhost:43217/__cypress/runner/cypress_runner.js:7608:14)\nFrom Your Spec Code:\n    at Context.eval (http://localhost:43217/__cypress/tests?p=cypress/integration/10.TestCoverage_spec.js:118:8)\n\nFrom Node.js Internals:\n  Error: Command failed: npm run test-coverage -- --coverageReporters=\"json-summary\" --testFailureExitCode=0\n  npm ERR! code ELIFECYCLE\n  npm ERR! errno 1\n  npm ERR! sd-0x-project-trybewallet@0.1.0 test-coverage: `react-scripts test --coverage --watchAll=false \"--coverageReporters=json-summary\" \"--testFailureExitCode=0\"`\n  npm ERR! Exit status 1\n  npm ERR! \n  npm ERR! Failed at the sd-0x-project-trybewallet@0.1.0 test-coverage script.\n  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n  \n  npm ERR! A complete log of this run can be found in:\n  npm ERR!     /home/gelso/.npm/_logs/2022-08-02T15_00_48_277Z-debug.log\n  \n      at ChildProcess.exithandler (child_process.js:294:12)\n      at ChildProcess.emit (events.js:198:13)\n      at maybeClose (internal/child_process.js:982:16)\n      at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)",
                "diff": null
              },
              "uuid": "7c0b6cc7-f36a-4107-8d5c-a03fcb7b989e",
              "parentUUID": "a39aed57-3607-4b66-b014-263b4adb40e5",
              "isHook": false,
              "skipped": false
            }
          ],
          "suites": [],
          "passes": [],
          "failures": [
            "31a7c7ad-1e00-4ebf-931b-0eb437e235ad",
            "dce3e8ac-5b7f-44f4-b1ef-30f95c59e3ce",
            "c39fe6f0-f625-4733-a58a-9e7f1745835d",
            "7c0b6cc7-f36a-4107-8d5c-a03fcb7b989e"
          ],
          "pending": [],
          "skipped": [],
          "duration": 16509,
          "root": false,
          "rootEmpty": false,
          "_timeout": 2000
        }
      ],
      "passes": [],
      "failures": [],
      "pending": [],
      "skipped": [],
      "duration": 0,
      "root": true,
      "rootEmpty": true,
      "_timeout": 2000
    }
  ],
  "meta": {
    "mocha": {
      "version": "7.0.1"
    },
    "mochawesome": {
      "options": {
        "quiet": false,
        "reportFilename": "mochawesome",
        "saveHtml": false,
        "saveJson": true,
        "consoleReporter": "spec",
        "useInlineDiffs": false,
        "code": true
      },
      "version": "7.1.3"
    },
    "marge": {
      "options": {
        "id": "default",
        "reportDir": "cypress/reports",
        "overwrite": false,
        "html": false,
        "json": true
      },
      "version": "6.2.0"
    }
  }
}